data/siddk.rb

```diff
@@ -127,20 +127,439 @@
+  author('Rico Jonschkowski and Oliver Brock'),
+  pages(407, 428),
+nil)
+
+entry!('lynch2019learning',
+  arxiv(2019, '1903.01973'),
+  title('Learning Latent Plans from Play'),
+  author('Corey Lynch and Mohi Khansari and Ted Xiao and Vikash Kumar and Jonathan Tompson and Sergey Levine and Pierre Sermanet'),
+nil)
+
+entry!('tzutalin2015labelimg',
+  title('LabelImg'),
+  author('Tzutalin'),
+  type('misc'),
+  howpublished('\url{https://github.com/tzutalin/labelImg}'),
+  year(2015),
+nil)
+
+entry!('openvino2018cvat',
+  title('Computer Vision Annotation Tool'),
+  author('OpenVinoToolKit'),
+  type('misc'),
+  howpublished('\url{https://github.com/openvinotoolkit/cvat}'),
+  year(2018),
+nil)
+
+
+
+########################################################################################################################
+###                        CS 330 Fall 2020 --> Meta-Learning for Sample Reweighting for NLI                         ###
+########################################################################################################################
+
+entry!('belinkov2019premise',
+  acl(2019),
+  title('Don\'t Take the Premise for Granted: Mitigating Artifacts in Natural Language Inference'),
+  author('Yonatan Belinkov and Adam Poliak and S. Shieber and Benjamin Van Durme and Alexander M. Rush'),
+nil)
+
+entry!('ren2018reweighting',
+  icml(2018),
+  title('Learning to Reweight Examples for Robust Deep Learning'),
+  author('Mengye Ren and Wenyuan Zeng and B. Yang and R. Urtasun'),
+nil)
+
+entry!('shu2019meta',
+  neurips(2019),
+  title('{Meta-Weight-Net}: Learning an explicit mapping for sample weighting'),
+  author('Jun Shu and Qi Xie and Lixuan Yi and Qian Zhao and Sanping Zhou and Zongben Xu and Deyu Meng'),
+  pages(1919, 1930),
+nil)
+
+entry!('kumar2010self',
+  neurips(2010),
+  title('Self-paced learning for latent variable models'),
+  author('M Pawan Kumar and Benjamin Packer and Daphne Koller'),
+  pages(1189, 1197),
+nil)
+
+entry!('ma2017self',
+  icml(2017),
+  title('Self-paced co-training'),
+  author('Fan Ma and Deyu Meng and Qi Xie and Zina Li and Xuanyi Dong'),
+nil)
+
+entry!('jiang2018mentornet',
+  icml(2018),
+  title('{MentorNet}: Learning data-driven curriculum for very deep neural networks on corrupted labels'),
+  author('Lu Jiang and Zhengyuan Zhou and Thomas Leung and Li-Jia Li and Li Fei-Fei'),
+  pages(2304, 2313),
+nil)
+
+entry!('chang2017active',
+  neurips(2017),
+  title('Active bias: Training more accurate neural networks by emphasizing high variance samples'),
+  author('Haw-Shiuan Chang and Erik Learned-Miller and Andrew McCallum'),
+  pages(1002, 1012),
+nil)
+
+entry!('grefenstette2019higher',
+  arxiv(2019, '1910.01727'),
+  title('Generalized Inner Loop Meta-Learning'),
+  author('Edward Grefenstette and Brandon Amos and Denis Yarats and Phu Mon Htut and Artem Molchanov and F. Meier and Douwe Kiela and Kyunghyun Cho and Soumith Chintala'),
+nil)
+
+entry!('nie2020anli',
```

Comment:
Would be nice not to change the name because that might break other people's bib...

Reply:
Fair - I changed it back. For the Levine2015 cite that I removed from your file (pliang.rb) I did the same, but the years are different... I went with the new year.